src/services/websockets/api/message.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Message types for the graasp websockets network API
+ * {@link https://github.com/graasp/graasp-plugin-websockets/blob/main/API.md}
+ */
+
+/**
+ * Message realms: describe the universe in which the message belongs to
+ */
+export enum Realms {
+  /** Notifications realm for real-time updates in Graasp */
+  Notif = 'notif',
+}
+
+export type Realm = Realms | `${Realms}`;
+
+/**
+ * Base message shape
+ */
+export interface Message {
+  realm: Realm;
+}
+
+/**
+ * Base message for real-time updates and notifications in Graasp
+ */
+export interface NotifMessage extends Message {
+  realm: `${Realms.Notif}`;
+}
+
+export * from './client';
+export * from './server';
```

Comment:
Why do you export these here ?

Reply:
I see what you mean, `Message` is the top-level interface of both `ClientMessage` and `ServerMessage`. I will import at the root then.